include/dev/scsi.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Based on FreeBSD `scsi_all.h`.
+ */
+#ifndef _SCSI_H_
+#define _SCSI_H_
+
+/* Opcodes. */
+#define REQUEST_SENSE 0x03
```

Comment:
We prefer enums since `gdb` can print their values as symbols. In this case maybe `typedef enum scsi_opcode { ... } scsi_opcode_t` ?

Reply:
I've decided to stick to what they do. If I changed these definitions, I'd have to change many definitions form other header files to stick to this convention (e.g. see `usb.h`).

Reply:
Ok.